src/lib/internal/actions/escape-keydown/action.ts

```diff
@@ -55,6 +55,8 @@ export const useEscapeKeydown = (node: HTMLElement, config: EscapeKeydownConfig
 			return;
 		}
 
+		e.preventDefault();
```

Comment:
This can't exist by itself without making changes elsewhere, as we're checking for default prevention elsewhere on this event, like [here](https://github.com/melt-ui/melt-ui/blob/bdf8bf25531b2c17bc9b0520b3300e0938e18577/src/lib/internal/actions/popper/action.ts#L79-L90) where if the event is `defaultPrevented` we _don't_ close the popper element.

Reply:
thanks! I noticed this but wasn't really sure if I can safely remove this check 😮‍💨

Reply:
If something breaks the test should fail, otherwise it's a great chance to identify which tests we're lacking! But after taking another look it appears that the link preview was the only one depending on it being handled there.

This is something we can address with #708.